src/components/Main.jsx

```diff
@@ -0,0 +1,198 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaw, faCirclePlus } from "@fortawesome/free-solid-svg-icons";
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import TodoItem from "./TodoItem";
+
+const Main = () => {
+  const initialData = localStorage.getItem("data")
+    ? JSON.parse(localStorage.getItem("data"))
+    : [];
+  const [data, setData] = useState(initialData);
+  useEffect(() => localStorage.setItem("data", JSON.stringify(data)), [data]);
+
+  const [value, setValue] = useState("");
+  const handleOnchange = (e) => {
+    setValue(e.target.value);
+  };
+
+  const handleSubmit = () => {
+    if (!value.trim()) {
+      setValue("");
+      return;
+    }
+    setData([
+      ...data,
+      {
+        id: new Date().getTime() + Math.random(),
+        content: value,
+        isDone: false,
+      },
+    ]);
+    setValue("");
+  };
+
+  const doneCount = data.filter((el) => el.isDone).length;
+  const totalCount = data.length;
+
+  const toggleTodo = (id) => {
+    const updatedData = data.map((el) =>
+      el.id === id ? { ...el, isDone: !el.isDone } : el
+    );
+    setData(updatedData);
+  };
+
+  const deleteTodo = (id) => {
+    const updatedData = data.filter((el) => el.id !== id);
+    setData(updatedData);
+  };
+
+  return (
+    <Wrapper>
+      <InputWrapper>
+        <FontAwesomeIcon icon={faPaw} size="lg" />
+        <Input
+          onKeyDown={(e) => {
+            e.key === "Enter" && handleSubmit();
+          }}
+          onChange={handleOnchange}
+          value={value}
+        />
```

Comment:
입력창에 할 일을 입력후 +버튼을 누르면 문제 없이 잘 랜더링 되는데, **엔터키를 눌렀을 때**는 **맨 마지막 글자가 2번 랜더링** 되고 있어요!
<br>
<img width="348" alt="스크린샷 2024-03-24 오전 12 07 36" src="https://github.com/CEOS-Developers/react-todo-19th/assets/117785708/2cdfd885-6ce3-4c6a-b3bc-1d62830da4ba">
1. `onKeyDown`을 `onKeyPress`로 바꾸거나
2. 밑에처럼 `event.nativeEvent.isComposing === false` 를 추가 
해보세용~
```suggestion
        <Input
           onKeyDown={(e) => {
               if (e.key === "Enter" && !e.nativeEvent.isComposing) {
               handleSubmit();
    }
  }}
           onChange={handleOnchange}
           value={value}
/>
```